giddy-banana/models/userModel.cjs

```diff
@@ -11,6 +11,10 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      models.User.belongsToMany(models.Article, { through: "Like_it" });
+      models.User.belongsToMany(models.Article, { through: "Recommend" });
+      models.User.belongsToMany(models.Article, { through: "Favourite" });
+      models.User.belongsToMany(models.Article, { through: "Comment" });
```

Comment:
What do you think about prepending the names with "User"? E.g.:
* UserLike
* UserRecommendation
* UserFavourite
* UserComment
or similar?